lua/gh-actions/github.lua

```diff
@@ -43,7 +44,11 @@ local function get_token_from_gh_cli(cmd)
   if cmd then
     res = vim.fn.system(cmd)
   else
-    res = vim.fn.system('gh auth token')
+    local gh_enterprise_flag = ""
+    if server ~= nil and server ~= "" then
+      gh_enterprise_flag = " --hostname " .. server
```

Comment:
I haven't done too much with shell commands and user(ish) input, but potentially we'd want to escape the server argument here.

```suggestion
      gh_enterprise_flag = " --hostname " .. vim.fn.shellescape(server)
```